acasclient/acasclient.py

```diff
@@ -2324,9 +2324,15 @@ def _get_meta_lot_by_parent_corp_name(self, parent_corp_name):
         """
         # Look up the compound to find a lot, so we can access a MetaLot
         search_results = self.cmpd_search(corpNameList=parent_corp_name)
```

Comment:
Corp name list has become a misnomer.  It's not an exact match search and we should fix this but not as part of this ticket.